sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 0.50.3 (unreleased)
 
+> **NOTE**: Starting with the GA release of version 7.0.0, (Currently in preview) this package will be deprecated.
```

Comment:
nitpick: should `(currently in preview)` be placed in front of the comma?

```suggestion
> **NOTE**: Starting with the GA release of version 7.0.0 (currently in preview), this package will be deprecated.
```

Reply:
In the Cosmos notes we changed "GA" -> "stable", as we weren't whether "GA" was too much of a corporate acronym...but up to you on this one.

Reply:
I think this (your suggestion) is technically more grammatically correct.  Nice, swapping.